File: functional_programming/fizzBuzz.scala
object FizzBuzzApp {
    def fizzBuzz(n: Int): String = {
        (n % 3, n % 5) match {
            case (0, 0) => "FizzBuzz"
            case (_, 0) => "Buzz"
            case (0, _) => "Fizz"
            case _      => n.toString
        }
    }
}

File: fizzBuzz.py
n = int(input())
def fib(n):
    if n % 15 == 0:
        return "FizzBuzz"
    elif n % 3 == 0:
        return "Fizz"
    elif n % 5 == 0:
        return "Buzz"
    else:
        return str(n)
list(map(lambda x : print(x), list(map(lambda x : fib(x), range(1, n+1)))))
